src/main/java/seedu/academyassist/logic/parser/ParserUtil.java

```diff
@@ -97,7 +98,7 @@ public static Subject parseSubject(String subject) throws ParseException {
         if (!Subject.isValidSubject(trimmedSubject)) {
             throw new ParseException(Subject.MESSAGE_CONSTRAINTS);
         }
-        return new Subject(trimmedSubject);
+        return new Subject(trimmedSubject.toUpperCase());
```

Comment:
Good that you consider different cases in which the input may be of different casing